pkg/services/librarypanels/database.go

```diff
@@ -62,3 +62,54 @@ func (lps *LibraryPanelService) deleteLibraryPanel(c *models.ReqContext, panelID
 
 	return err
 }
+
+// getLibraryPanels gets Library Panels based on statement and args
```

Comment:
Why "Library Panels", if it's not referring to the LibraryPanel type? I'd think that if you're not referring to the specific type, you'd just write library panels?

Reply:
Fixed